src/modules/proxy-deployer/tests/proxy-deployer.abi.service.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('Proxy  Deployer Abi Service', () => {
         sender: 'erd1dc3yzxxeq69wvf583gw0h67td226gu2ahpk3k50qdgzzym8npltq7ndgha',
         signature: undefined,
         value: '0',
-        version: 1,
+        version: 2,
```

Comment:
Just a double check - are there third party consumers of the NFT service that take crafted transactions and relay them? If so, if the construction of relayed V2 transactions on their side is done with old / obsolete components, they might encounter some issues (old / obsolete builders of relayed V2 do not know to use the `innerTx.version`).

Reply:
no.. there are not, as far as I know